### PR TITLE
update covidcast sources api docs

### PR DESCRIPTION
**Pull request by pedritom-amzn**
update documentation for `covidcast` and `covidcast_meta` endpoints. update project `README` to link to api docs. switch licensing wording in `dengue_nowcast` docs to delphi-owned language.

### CODE REVIEW

File: docs/api/covidcast_meta.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ None.
 | --- | --- | --- |
 | `result` | result code: 1 = success, 2 = too many results, -2 = no results | integer |
 | `epidata` | list of results, one per name/geo_type pair | array of objects |
-| `epidata[].name` | Data souce, and subtype if applicable (e.g. fb_survey_cli, fb_survey_ili) | string |
+| `epidata[].source` | Data souce, and subtype if applicable (e.g. fb_survey_cli, fb_survey_ili) | string |
```
Comment (pedritom-amzn) [inline docs/api/covidcast_meta.md:61]: i'm fixing the description to just say "data source"

File: docs/api/covidcast_meta.md [hunk 2]
```diff
@@ -74,51 +74,17 @@ https://delphi.midas.cs.cmu.edu/epidata/api.php?source=covidcast_meta
   "result": 1,
   "epidata": [
     {
-      "name": "fb_survey-cli",
+      "source": "fb_survey",
```
Comment (pedritom-amzn) [inline docs/api/covidcast_meta.md:77]: i'm updating this to "data_source".

File: docs/api/covidcast_meta.md [hunk 3]
```diff
@@ -58,7 +58,7 @@ None.
 | --- | --- | --- |
 | `result` | result code: 1 = success, 2 = too many results, -2 = no results | integer |
 | `epidata` | list of results, one per name/geo_type pair | array of objects |
-| `epidata[].name` | Data souce, and subtype if applicable (e.g. fb_survey_cli, fb_survey_ili) | string |
+| `epidata[].source` | Data souce, and subtype if applicable (e.g. fb_survey_cli, fb_survey_ili) | string |
 | `epidata[].geo_type` | geographic resolution (e.g. `county`, `hrr`, `msa`, `dma`, `state`) | string |
 | `epidata[].min_date` | minimum date (yyyy-MM-dd) | string |
 | `epidata[].max_date` | maximum date (yyyy-MM-dd) | string |
```
Comment (pedritom-amzn) [inline docs/api/covidcast_meta.md:64]: looks like i neglected to change these to "min/max_time" and to add "min/max_value". i'll do that now.

File: docs/api/covidcast_meta.md (hunk 1 above)
Comment (undefx) [reply docs/api/covidcast_meta.md:61]: thanks! also in this table, do you mind (if you have the time) adding: `time_type`, `min_value`, and `max_value`? also, `min_date` -> `min_time` and `max_date` -> `max_time` (with data type always int, YYYYMMDD for day, YYYYWW for week)

Comment (pedritom-amzn) [reply docs/api/covidcast_meta.md]: fixed

File: docs/api/covidcast_meta.md (hunk 2 above)
Comment (pedritom-amzn) [reply docs/api/covidcast_meta.md]: fixed

File: docs/api/covidcast_meta.md (hunk 3 above)
Comment (pedritom-amzn) [reply docs/api/covidcast_meta.md]: fixed

**Comment by pedritom-amzn** [comment]
thanks, feedback addressed.

**Pull request merged**